armi/reactor/components/__init__.py

```diff
@@ -118,9 +118,13 @@ class UnshapedComponent(Component):
     A component with undefined dimensions.
 
     Useful for situations where you just want to enter the area directly.
-    For instance, in filler situations where the exact shape of this component is
-    is unknown but you have some left-over space between other components filled
-    with a known material you might need to model.
+
+    For instance, when you want to model neutronic behavior of an assembly based
+    on only knowing the area fractions of each material in the assembly.
+
```

Comment:
I'd consider even taking this a step further:
1. the sum of the areal fractions of unshaped components need to be equal with the total area of the block. (but maybe there are other places this could/should go)
2. expanding on the neutronics argument that this type is useful for homogenization-like studies when particular shapes/locations of components is not important (since they end up getting smeared together anyway) but rather knowing the area (and ultimately the volume) of a material within a block and how it can change with temperature.

Just a thought.

Reply:
Sorry, I've already merged the PR.  But @keckler , do you have any thoughts here?